spec/Section 6 -- Execution.md

```diff
@@ -34,12 +34,18 @@ ExecuteRequest(schema, document, operationName, variableValues, initialValue):
     * Return {ExecuteQuery(operation, schema, coercedVariableValues, initialValue)}.
   * Otherwise if {operation} is a mutation operation:
     * Return {ExecuteMutation(operation, schema, coercedVariableValues, initialValue)}.
+  * Otherwise if {operation} is a subscription operation:
+    * Return {Subscribe(operation, schema, coercedVariableValues, initialValue)}.
 
 GetOperation(document, operationName):
 
   * If {operationName} is {null}:
     * If {document} contains exactly one operation.
-      * Return the Operation contained in the {document}.
+      * Let {operation} be the Operation contained in the {document}.
+      * If {operation} is a subscription operation:
+        * If {operation} contains more than one root field, produce a query error.
+        * Return {operation}.
+      * Otherwise return {operation}.
```

Comment:
These new steps aren't necessary. The validation rule will check for this, and the `CreateSourceEventStream` already does the right thing of using the first (only) field in the set